assets/translations/en.json

```
@@ -46,12 +48,15 @@
     "fees": "Fees",
     "recipientAddress": "Recipient address",
     "transactionHash": "Transaction Hash",
+    "transactionSuccessful": "Transaction Successful",
```

Biome (1.9.4)

[error] The key transactionSuccessful was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)